app/src/main/java/io/homeassistant/companion/android/widgets/common/WidgetAuthenticationActivity.kt

```diff
@@ -15,12 +15,20 @@ class WidgetAuthenticationActivity : AppCompatActivity() {
         private const val TAG = "WidgetAuthenticationA"
     }
 
+    private var authenticating = false
+
     override fun onCreate(savedInstanceState: Bundle?) {
         super.onCreate(savedInstanceState)
         Log.d(TAG, "onCreate in WidgetAuthenticationActivity called")
+    }
 
-        val authenticator = Authenticator(this, this, ::authenticationResult)
-        authenticator.authenticate(getString(R.string.biometric_set_title))
+    override fun onWindowFocusChanged(hasFocus: Boolean) {
```

Comment:
should we consider adding a log statement here for consistency purposes in the file?

Reply:
That feels like a log statement you only use during debugging. If it was up to me, the one in `onCreate` is also unnecessary.